woocommerce-services.php

```diff
@@ -9,7 +9,7 @@
  * Domain Path: /i18n/languages/
  * Version: 1.26.1
  * WC requires at least: 3.5.5
- * WC tested up to: 6.4
+ * WC tested up to: 6.5.1
```

Comment:
I tested this on WC 6.6.1 and it seems still working. Should we bump this to 6.6.1?